### ISSUE TITLE
dasc2_filter --build is looking for non-existing file ("build_versions.json")

### ISSUE THREAD

**Issue by robbernan** [davidkuhta/dasc2#2]
When running following command `dasc2_filter --build 55958`, I get such response:

```
Traceback (most recent call last):
  File "/Users/XXX/anaconda/bin/dasc2_filter", line 11, in <module>
    sys.exit(main())
  File "/Users/XXX/anaconda/lib/python3.6/site-packages/dasc2/bin/filter_replays.py", line 170, in main
    processed_version = int(check_build_version(args.vers, False))
  File "/Users/XXX/anaconda/lib/python3.6/site-packages/dasc2/lib/replay_helpers.py", line 87, in check_build_version
    version_file = open(resource_filename(__name__, '../ref/build_versions.json')).read()
FileNotFoundError: [Errno 2] No such file or directory: '/Users/XXX/anaconda/lib/python3.6/site-packages/dasc2/lib/../ref/build_versions.json'
```

**Comment by robbernan** [comment]
Files in ref folder were not downloaded when pip was used to install the package.

**Comment by davidkuhta** [comment]
Hi @robbernan,
Thanks for submitting this issue; as you determined it was a bug in the package being uploaded to `pip` vs the repo contents.
Most of our development had been using Python 2, so I spent some time refactoring the code to more succinctly support Python 3 as well.
I did a test run of the newest package I uploaded to `pip` with your same setup of Anaconda & Python 3.6 and it ran smoothly, so feel free to give that a try and open another issue if you run into anything else.
As an aside, welcome to GitHub and happy holidays!
